server/app.js

```diff
@@ -194,7 +198,7 @@ module.exports = function createApp({
 
     // CSRF protection
     if (!testMode) {
-        app.use(csurf());
+        app.use(csurf({cookie: true}));
```

Comment:
Seems like we are already using this above

Reply:
Good spot!